stubs/qrcode/qrcode/base.pyi

```diff
@@ -1,25 +1,27 @@
-from _typeshed import Incomplete
-from typing import NamedTuple
+from collections.abc import Iterator
+from typing import NamedTuple, SupportsIndex
 
-EXP_TABLE: Incomplete
-LOG_TABLE: Incomplete
-RS_BLOCK_OFFSET: Incomplete
-RS_BLOCK_TABLE: Incomplete
+from ._types import ErrorCorrect
 
-def glog(n): ...
-def gexp(n): ...
+EXP_TABLE: list[int]
+LOG_TABLE: list[int]
+RS_BLOCK_OFFSET: dict[ErrorCorrect, int]
+RS_BLOCK_TABLE: tuple[tuple[int, int, int] | tuple[int, int, int, int, int, int], ...]
```

Comment:
I haven't looked at it, but I assume these could be made `Final`?

Reply:
These should indeed be made Final, as does the `LUT.rsPoly_LUT` table. I've done so now.